src/connectors/search-box/connectSearchBox.js

```diff
@@ -63,7 +64,7 @@ const withUsage = createDocumentationMessageGenerator({
  *   })
  * );
  */
-export default function connectSearchBox(renderFn, unmountFn) {
+export default function connectSearchBox(renderFn, unmountFn = noop) {
```

Comment:
Is this consistent with the other connectors?

Reply:
For some of them - Yes. It should be the norm though. Otherwise, the connector throws an error on `dispose`. Hence the test on each PR I've opened about the `dispose` function.